ProgramowanieBot/Modules/ApplicationCommands/SlashCommands/ResolveCommand.cs

```diff
@@ -25,30 +27,20 @@ public class ResolveCommand(IServiceProvider serviceProvider, IOptions<Configura
         User? helper2 = null)
     {
         var configuration = options.Value;
-
-        var channel = Context.Channel;
-        var channelId = channel.Id;
+        var channelId = Context.Channel.Id;
 
         await using (var context = serviceProvider.GetRequiredService<DataContext>())
             if (await context.Posts.AnyAsync(p => p.PostId == channelId && p.IsResolved))
                 throw new(configuration.Interaction.PostAlreadyResolvedResponse);
 
-        await Context.Client.Rest.SendMessageAsync(configuration.Interaction.PostResolvedNotificationChannelId, $"**{string.Format(configuration.Interaction.PostResolvedNotificationMessage, channel)}**");
+        await PostsHelper.SendPostResolveMessagesAsync(channelId, Context.User.Id, helper.Id, helper2?.Id, Context.Client.Rest, configuration);
 
-        var isHelper2 = helper2 != null && helper != helper2;
-        var user = Context.User;
         return InteractionCallback.Message(new()
         {
-            Content = $"**{configuration.Emojis.Success} {(isHelper2 ? string.Format(configuration.Interaction.WaitingForApprovalWith2HelpersResponse, helper, helper2) : string.Format(configuration.Interaction.WaitingForApprovalResponse, helper))}**",
-            Components =
-            [
-                new ActionRowProperties(
-                [
-                    new ActionButtonProperties($"approve:{helper.Id}:{helper != user}:{(isHelper2 ? helper2!.Id : null)}:{(isHelper2 ? helper2 != user : null)}", configuration.Interaction.ApproveButtonLabel, ButtonStyle.Success),
-                ]),
-            ],
-            AllowedMentions = AllowedMentionsProperties.None,
+            Content = configuration.Emojis.Success,
+            Flags = MessageFlags.Ephemeral
         });
+
```

Comment:
Isn't it better to send the message about resolve in the interaction response instead of responding with an emoji and sending a normal message?
You also added an unnecessary blank line at the end.

Reply:
One of the main points of this PR, besides moving the Approve button, was to change this exact behaviour that was adapted earlier. I wanted to get rid of the ugly text that said "Original message was deleted" (as seen on the screenshot). 
![image](https://github.com/Programowanie-Discord/ProgramowanieBot/assets/64475360/899ad5b3-3e29-4302-a0f0-76695587252b)
Naturally, when it comes to using a slash command this does not take place, however I wanted to maintain some sort of consistence and make all the Post Resolve messages regular messages instead of replies or interaction responses.

Also, thank you for your additional feedback regarding my code formatting. I will include that in my next commit.

Reply:
As promised, formatting issue fixed with 2eee47c.

Reply:
But I think slash commands look good. "Original message was deleted" happens only when using a component of an ephemeral message.

Reply:
I am aware of that, but, again, all I wanted was a bit of consistency. Moreover, changing that would require either changing the helper method or not using it, which would require more code while the change wouldn't be neither that noticeable nor important for the user.

Reply:
The whole change won't be noticeable by users at all but I think sending an interaction response with an emoji is not a good idea, it's better to just respond to the interaction with the resolve message. We have more inconsistency with the current approach either.

Reply:
So what are you suggesting? Abandoning the helper in this particular case, or adding an exception to the method where if it's a slash command it will use an interaction response rather than a regular message?